contentcuration/contentcuration/models.py

```diff
@@ -43,7 +49,7 @@ class TopicTree(models.Model):
         help_text=_("For different versions of the tree in the same channel (trash, edit, workspace)"),
     )
     root_node = models.ForeignKey(
-        'Node',
+        'ContentMetadata',
```

Comment:
When it comes to converting content curation to content app nodes, will having the same name complicate the plugin to do this?